### ISSUE TITLE
catkin_make error with libtm.a

### ISSUE THREAD

**Issue by xs-alt** [realsenseai/realsense-ros#502]
Hi all
   I am in Ubuntu 16.04, ROS Kinetic

CMake Error at /usr/lib/x86_64-linux-gnu/cmake/realsense2/realsense2Targets.cmake:84 (message):
  The imported target "realsense2::tm" references the file

     "/usr/lib/x86_64-linux-gnu/libtm.a"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/usr/lib/x86_64-linux-gnu/cmake/realsense2/realsense2Targets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/realsense2/realsense2Config.cmake:33 (include)
  realsense/realsense2_camera/CMakeLists.txt:35 (find_package)

l really do not find libtm.a in my computer and others, do not know what causes the problem.

**Comment by ArjunBalakrishnan** [comment]
Hello all,

I also have this same issue. I remember following the same installation tutorial few months back on a similar environment and it worked well. Does it have anything to do with new release?  Any help?

Thank you.

**Comment by mistoll** [comment]
I have the same issue

A downgrade of librealsense-dev from 2.16.2 to 2.16.1 solves the issue.
```
sudo apt-get install librealsense2-dev=2.16.1-0~realsense0.88
```

**Comment by taochenshh** [comment]
I am having the same issue.

**Comment by varandaas** [comment]
The @mistoll workaround worked for me. For a clean installation, follow these steps:
```
version="2.16.1-0~realsense0.88"
sudo apt-get install librealsense2-dkms
sudo apt install librealsense2=${version}
sudo apt-get install librealsense2-utils=${version}  
sudo apt-get install librealsense2-dev=${version}
sudo apt-get install librealsense2-dbg=${version}
```

**Comment by taochenshh** [comment]
@varandaas , @mistoll thanks, it worked

**Comment by taochenshh** [comment]
@varandaas @mistoll can you receive image data after fixing the catkin_make error, I cannot receive any image data as described in #503 .

**Comment by varandaas** [comment]
Affirmative @taochenshh, I have image

**Comment by diegomrt** [comment]
Nice, thanks @varandaas and @mistoll, it also worked for me!

**Comment by doronhi** [comment]
Work on a new release for the realsense2_camera to match the new librealsense 2.16.2 is in progress...

**Comment by xs-alt** [comment]
@doronhi  
Thanks
Is the issue also related to the problem described in #503 ?

**Comment by doronhi** [comment]
The problem was with librealsense v2.16.2. Fixed in v2.16.3
